javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,64 @@
+// Age-ify (A future age calculator)
+
+// let yearOfBirth = 1995
```

Comment:
So much commented code, maybe it is a mistake?

Please uncommment the code that is needed for review and let's try again 👍

Reply:
I was do that for easy to read console. I forget to fixed that. I fixed codes from master.